src/Billable.php

```diff
@@ -184,13 +201,11 @@ public function invoice()
     {
         if ($this->stripe_id) {
             try {
-                StripeInvoice::create(['customer' => $this->stripe_id], $this->getStripeKey())->pay();
+                return StripeInvoice::create(['customer' => $this->stripe_id], $this->getStripeKey())->pay();
             } catch (StripeErrorInvalidRequest $e) {
                 return false;
             }
         }
-
-        return true;
```

Comment:
Suggestions? I'd like to be able to get back the invoice object from Stripe, and returning the object from Stripe would not be considered false.